doc/workflows_and_tools/tools/index.rst

```diff
@@ -31,6 +31,281 @@ a glimpse in the connection to your user store, you could as well in a first
+....................................
+
+Interpret tokeninfo-values as integers and match only if they are greater than the given integer.
+
+Example::
+
+    privacyidea-token-janitor find --tokeninfo-key timeWindow --tokeninfo-value-greater-than 100
+
+Search for all tokens with the tokeninfo-key ``timeWindow`` and the associated tokeninfo-value greater than ``100``.
+
+Actions
+*******
+
+Actions that can be performed on the found tokens.
+
+mark - disable - delete - unassign - export - listuser - tokenrealms
+
+mark
+....
+
+**Mark** makes it possible to mark single or multiple tokens in order to carry out further actions with them later.
+
+Please note that without a previous selection of a certain token, all found tokens are automatically marked.
+
+Example::
+
+    privacyidea-token-janitor find --serial OATH0004C934 --action mark
+
+The token with the serial ``OAUTH0004C934`` will be marked.
+
+
+disable
+.......
+
+With **disable** single or multiple tokens can be disabled.
+
+Please note that without a previous selection of a certain token, all found tokens will be disabled.
+
+Example::
+
+    privacyidea-token-janitor find --serial OATH0004C934 --action disable
+
+The token with the serial ``OAUTH0004C934`` will be disabled.
+
+delete
+......
+
+With **delete** single or multiple tokens can be deleted.
+
+Please note that without a previous selection of a certain token, all found tokens will be deleted.
+
+Example::
+
+    privacyidea-token-janitor find --serial OATH0004C934 --action delete
+
+The token with the serial ``OAUTH0004C934`` will be deleted.
+
+export
+......
+
+With **export** single or multiple tokens can be exported as csv or pskc.
+
+Please note that without a previous selection of a certain token, all found tokens will be exported.
+
+Export only possible with HOTP and TOTP token.
+
+Example::
+
+    privacyidea-token-janitor find --serial OATH0004C934 --action export > OAUTH0004C934.xml
+
+The token with the serial ``OAUTH0004C934`` will be exported and saved in an xml file.
+
+Note that you need your encryption key for re-import.
+
+listuser
+........
+
+With **listuser** the various users and their tokens are listed in a summarized view.
+
+Example::
+
+    privacyidea-token-janitor find --action listuser
+
+lists all users and their tokens together.
+
+sum
+___
+
+**Sum** and **listuser** together list all users with their information and show how many tokens each user has assigned.
+
+Example::
+
+    privacyidea-token-janitor find --sum --action listuser
+
+tokenrealms
+...........
+
+**Tokenrealms** can be used to assign tokens to different realms.
+
+To do this, the ``tokenrealms`` function is also required.
+
+Please note that without a previous selection of a certain token, all found tokens will be assigned to the realm.
+
+Example::
+
+    privacyidea-token-janitor find --serial OATH0005B88E --action tokenrealms --tokenrealms defrealm
+
+Setting realms of token ``OATH0005B88E`` to ``defrealm``.
```

Comment:
You can also assign a list or realms by comma separating:

    privacyidea-token-janitor find --serial OATH0005B88E --action tokenrealms --tokenrealms defrealm,realmA,realmB